app/components/Input/input.css

```diff
@@ -24,11 +24,15 @@
 /* size */
 
 .input.large {
-  height: var(--spacing-56);
+  height: 56px;
   font-size: 18px;
   letter-spacing: -0.18px;
 }
 
+.input.shadow {
+  box-shadow: 0px 16px 40px 0px rgba(175, 183, 194, 0.2);
```

Comment:
We can keep this for now, but general note is I wonder... If there's a way to just reference the same CSS that we already created for shadows. Like by referencing that class in the component itself. I.e. all input components have the class "shadow" (or whatever it's called) that we already defined. Ideally we would have a system where CSS never has to be written twice, to keep things perfectly consistent! 

@mruwnik could you comment on the feasibility of this?

Reply:
I mean you can of course create a .shadow class in the root.css file and reference it to any component if that is what you mean. I just haven't been doing that because you said to not mess with the root.css file

Reply:
I added a `shadowed` class to root.css that does this

Reply:
I'm sorry, amendment: you can add your own classes to root.css, particularly one-off classes. Just don't add to or edit the classes I already made heh.

Reply:
But yes from now on reference Nemo's shadowed class!